src/eko/anomalous_dimensions/as4/__init__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+This file contains the |N3LO| Altarelli-Parisi splitting kernels.
+
+
```

Comment:
This section is fine, but I would explicitly add a page in the Sphinx docs, providing verbose details about the parametrization (that are easier to read and understand than the notebook for someone else, including our future selves, and we'll reuse for the paper, or just cite).

Once it's available, I'd add a pointer here.